engine/CMakeLists.txt

```diff
@@ -126,7 +126,6 @@ IF (CMAKE_SYSTEM_NAME STREQUAL Darwin)
 ENDIF ()
 
 SET(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
-SET(CMAKE_INSTALL_UCRT_LIBRARIES TRUE)
```

Comment:
This is about the only consequential change in this PR, AFAIK.

Reply:
OK, now there are more changes. All in this same file though.

Reply:
wouldn't this install the Visual C Runtime Libraries we need?

Reply:
Nope. This line installed the ucrt* and api-ms-* DLLs, which are separate.